compose-jetsnack/android/build.gradle.kts

```diff
@@ -26,8 +26,11 @@ android {
         versionName = "1.0-SNAPSHOT"
     }
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_17
```

Comment:
Does android app compile with this change?

Reply:
@eymar yep, this was actually the reason why it's done - otherwise build complained to the inconsistensies of versions for some reason